api_tests/comments/views/test_comment_report_list.py

```diff
@@ -237,10 +236,10 @@ def test_public_node_non_contributor_can_report_comment(self):
             comment (comment_level == 'public), non-contributors
             can also report comments.
         """
-        project = ProjectFactory(is_public=True, comment_level='public')
```

Comment:
This is now testing the same thing as `test_public_node_logged_in_non_contributor_can_report_comment`. In this test, can you instead make `comment_level = private` and test that non-contributors cannot report comments on public projects with a private comment level? Basically what `test_public_node_logged_in_non_contributor_cannot_report_comment` was doing.